packages/ember-runtime/tests/computed/reduce_computed_macros_test.js

```diff
@@ -653,38 +668,43 @@ QUnit.test('it throws an error if given fewer or more than two dependent propert
 QUnit.test('it has set-diff semantics', function() {
   var array1 = get(obj, 'array');
   var array2 = get(obj, 'array2');
-  var diff = get(obj, 'diff');
 
-  deepEqual(diff, [1, 2, 6, 7], 'set-diff is initially correct');
+  deepEqual(sortedGet(obj, 'diff'), [1, 2, 6, 7], "set-diff is initially correct");
 
   run(function() {
     array2.popObject();
   });
-  deepEqual(diff, [1,2,6,7], 'removing objects from the remove set has no effect if the object is not in the keep set');
+
+  deepEqual(sortedGet(obj, 'diff'), [1,2,6,7], "removing objects from the remove set has no effect if the object is not in the keep set");
 
   run(function() {
     array2.shiftObject();
   });
-  deepEqual(diff, [1, 2, 6, 7, 3], 'removing objects from the remove set adds them if they\'re in the keep set');
+
+  deepEqual(sortedGet(obj, 'diff'), [1, 2, 3, 6, 7], "removing objects from the remove set adds them if they're in the keep set");
 
   run(function() {
     array1.removeObject(3);
   });
-  deepEqual(diff, [1, 2, 6, 7], 'removing objects from the keep array removes them from the computed array');
+
+  deepEqual(sortedGet(obj, 'diff'), [1, 2, 6, 7], "removing objects from the keep array removes them from the computed array");
 
   run(function() {
     array1.pushObject(5);
   });
-  deepEqual(diff, [1, 2, 6, 7], 'objects added to the keep array that are in the remove array are not added to the computed array');
+
+  deepEqual(sortedGet(obj, 'diff'), [1, 2, 6, 7], "objects added to the keep array that are in the remove array are not added to the computed array");
 
   run(function() {
     array1.pushObject(22);
   });
-  deepEqual(diff, [1, 2, 6, 7, 22], 'objects added to the keep array not in the remove array are added to the computed array');
+
+  deepEqual(sortedGet(obj, 'diff'), [1, 2, 6, 7, 22], "objects added to the keep array not in the remove array are added to the computed array");
 });
 
 
 function commonSortTests() {
+<<<<<<< HEAD
```

Comment:
Ya, im just going through the work to re-integrate and fix this. Figured I would get it as a PR sooner rather then later so others saw this was being worked on.